test/opt/strength_reduction_test.cpp

```diff
@@ -63,36 +63,45 @@ TEST_F(StrengthReductionBasicTest, BasicReplaceMulBy8) {
   EXPECT_THAT(output, HasSubstr("OpShiftLeftLogical %uint %uint_5 %uint_3"));
 }
 
-// Test to make sure we replace 16*5.
+// TODO(dneto): Add Effcee as requried dependency, and make this unconditional.
```

Comment:
s/requried/required/

Reply:
Also, I'm wondering if we could somehow make the harness execute tests marked as EFFCEE so that the test itself can run unconditionally when EFFCEE is enabled.  If that's too annoying, that's fine.  Adding #ifdefs is certainly not a big deal.

Reply:
Oh, and of course, our bots now have to run with and without EFFCEE enabled...

Reply:
My plan is to do a 3-step soft launch:
- add as conditional dependency now
- notify downstream projects so they can react, e.g. Android NDK, and LunarG
- make it a required dependency

If it's just a few weeks between steps 1 and 3 then I think we can skip running bots having Effcee off.